deploy-board/deploy_board/templates/environs/env_landing.html

```diff
@@ -513,7 +513,7 @@ <h4 class="panel-title pull-left pointer-cursor">
         </script>
         <div class="collapse in panel-body">
             <div id="activeDeployId">
-                {% include "deploys/deploy_progress.tmpl" %}
+                {% include "deploys/deploy_progress.html" %}
```

Comment:
Why the rename? It's not really html.

Reply:
Found on the Django docs:

> A template is a text file. It can generate any text-based format (HTML, XML, CSV, etc.). A template contains variables, which get replaced with values when the template is evaluated, and tags, which control the logic of the template.

https://docs.djangoproject.com/en/5.0/ref/templates/language/#templates

Reply:
tmpl files (Django extension installed):

<img width="701" alt="image" src="https://github.com/pinterest/teletraan/assets/72234714/1813159c-d69a-44a4-95c4-8b5a935daacd">

html files:

<img width="823" alt="image" src="https://github.com/pinterest/teletraan/assets/72234714/70b9e957-1f09-4a4a-ae9d-1a4655d86520">

Reply:
SGTM